### ISSUE TITLE
Assembled file isn't loaded ...

### ISSUE THREAD

**Issue by BacchusFLT** [Swoffa/SublimeKickAssemblerC64#33]
One want that pressing F7, the project should assemble, launch VICE and load the recently assembled file. However introducing segments, this doesn't work.

When I assemble the below WITH the ".segment" row, address 1000 has the labels
 in the monitor, but not the actual code. The binary was clearly never loaded. Commenting out the .segment it's loaded as normal.

```
	.segment MyCode[]

	*=$1000
start:	inc $d020
	jmp start
```

**Comment by Swoffa** [comment]
Thanks for reporting, will look into it.

**Comment by Swoffa** [comment]
@BacchusFLT 
My suggested solution is to put annotations in the file being built, something like thins:
```
// @kickass-build: {"file-to-run": "file.prg", "disk-to-run": "disk.d64"} 
	.segment MyCode[]

	*=$1000
start:	inc $d020
	jmp start
```
The `file-to-run`-annotation sets which prg-file to run in vice (or C64Debugger) and the `disk-to-run`-annotation sets which d64 to load into Vice. If `disk-to-run` is set, `file-to-run` is loaded from that disk, otherwise from the file system.

Would this solution give you what you need?

**Comment by BacchusFLT** [comment]
I wasn't aware of this option. Thx!

/Pontus Berg
 Bergatrollet AB
 CEO & Owner
 Tel/SMS: +46 735 082860
 Skype: BacchusBerg
 Hangout: pontus.bergatrollet@gmail.com
 www.bergatrollet.se


Den mån 10 dec. 2018 kl 09:03 skrev Simon Oskarsson <
notifications@github.com>:

**Comment by BacchusFLT** [comment]
Yes I guess it would.

/Pontus Berg
 Bergatrollet AB
 CEO & Owner
 Tel/SMS: +46 735 082860
 Skype: BacchusBerg
 Hangout: pontus.bergatrollet@gmail.com
 www.bergatrollet.se


Den mån 10 dec. 2018 kl 11:29 skrev Simon Oskarsson <
notifications@github.com>:

**Comment by Swoffa** [comment]
Fixed in PR #50. Will be in release 1.3.1.
